scipy/linalg/matfuncs.py

```diff
@@ -668,3 +675,329 @@ def rounded_sign(x):
         return S0
     else:
         return S0, errest
+
+
+def is_matrix_hermitian(A):
+    """
+    Returns True if a matrix is hermitian
+    Parameters
+    ----------
+    A : (N, N) array_like
+        Matrix to evaluate.
+    
+    Returns
+    -------
+    X : boolean
+        True if a matrix A is hermitian.
+
+    Examples
+    --------
+    >>> from scipy.linalg import is_matrix_hermitian
+    >>> m = np.array([[0,1j],[-1j,0]])
+    >>> print(is_matrix_hermitian(m))
+    True
+    >>> m = np.array([[5,0],[1,3]])
+    >>> print(is_matrix_hermitian(m))
+    False
+    """
+    A = _asarray_square(A)
+    return np.sum(A == A.conj().T) == (A.shape[1]**2)
+
+
+def is_matrix_symmetric(A):
```

Comment:
`is_symmetric()`

Reply:
Same as hermitian.

Reply:
it should be with single underscore not two

Reply:
Fixed